### PR TITLE
CHS Translation

### PR DESCRIPTION
**Pull request by WuzgXY-GitHub**
@TUsama I wonder if you can help me review this ..(:з」∠)..

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/zh_cn.json [hunk 1]
```diff
@@ -45,25 +49,36 @@
   "item.farmersdelight.golden_knife": "金刀",
   "item.farmersdelight.diamond_knife": "钻石刀",
 
-  "item.farmersdelight.mixed_salad": "混合沙拉",
   "item.farmersdelight.barbecue_stick": "烧烤串",
+  "item.farmersdelight.egg_sandwich": "鸡蛋三明治",
   "item.farmersdelight.chicken_sandwich": "鸡肉三明治",
   "item.farmersdelight.hamburger": "汉堡包",
+  "item.farmersdelight.dumplings": "饺子",
+  "item.farmersdelight.stuffed_potato": "填馅马铃薯",
```
Comment (TUsama) [inline src/main/resources/assets/farmersdelight/lang/zh_cn.json:57]: |Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

酿

File: src/main/resources/assets/farmersdelight/lang/zh_cn.json [hunk 2]
```diff
@@ -90,6 +105,13 @@
   "farmersdelight.advancement.craft_stove.desc": "得到一个炉灶",
   "farmersdelight.advancement.place_cooking_pot": "晚餐上桌喽！",
   "farmersdelight.advancement.place_cooking_pot.desc": "放置一个厨锅并准备肉！",
+  "farmersdelight.advancement.master_chef": "厨神当道",
+  "farmersdelight.advancement.master_chef.desc": "吃一顿包含每一道可用的菜的大餐！",
```
Comment (TUsama) [inline src/main/resources/assets/farmersdelight/lang/zh_cn.json]: |Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

> Eat a course of every meal available!

将全部的菜肴品尝一遍！

available一般不这么翻译，要不翻译成“可以...的”，要不就直接不翻译。
这里存疑，可以问一下Ex

**Review by TUsama — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/farmersdelight/lang/zh_cn.json (hunk 2 above)
Comment (WuzgXY-GitHub) [reply src/main/resources/assets/farmersdelight/lang/zh_cn.json]: 好的。
@Determancer

Comment (WuzgXY-GitHub) [reply src/main/resources/assets/farmersdelight/lang/zh_cn.json]: 我个人的意见是，可用指的是没被删除的菜，但我也不确定，你的想法很明显比我好。我翻译跌势后也是感到困惑的。

File: src/main/resources/assets/farmersdelight/lang/zh_cn.json (hunk 1 above)
Comment (WuzgXY-GitHub) [reply src/main/resources/assets/farmersdelight/lang/zh_cn.json:57]: Resolve

Comment (WuzgXY-GitHub) [reply src/main/resources/assets/farmersdelight/lang/zh_cn.json:57]: |Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

Resolve

Comment (WuzgXY-GitHub) [reply src/main/resources/assets/farmersdelight/lang/zh_cn.json:57]: |Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

Resolve

File: src/main/resources/assets/farmersdelight/lang/zh_cn.json (hunk 2 above)
Comment (WuzgXY-GitHub) [reply src/main/resources/assets/farmersdelight/lang/zh_cn.json]: 此外，别光看我改了的东西，麻烦也看看其它的部分如何，也是我之前翻译的。

**Review by TUsama — APPROVED** [review]

**Review by vectorwing — APPROVED** [review]
Really cool to see your reviewing process! All I'm able to do is pass some lines through Google Translate, but from what I saw, the updated lines do fit better.

I'll merge it now, if that's okay with you! 😄

**Comment by WuzgXY-GitHub** [comment]
Yes you can merge it now :)

**Pull request merged**